zebra-network/src/meta_addr.rs

```diff
@@ -458,6 +458,25 @@ impl MetaAddr {
         }
     }
 
+    /// Has this peer been seen recently?
+    ///
+    /// Returns `true` if this peer has responded recently or if the peer was gossiped with a
+    /// recent reported last seen time.
+    ///
+    /// [`constants::REACHABLE_PEER_DURATION`] represents the maximum time since a peer was seen to
+    /// still be considered reachable.
+    pub fn was_recently_reachable(&self) -> bool {
+        if let Some(last_seen) = self.last_seen() {
+            // Correctness: `last_seen` shouldn't ever be in the future, either because we set the
+            // time or because another peer's future time was sanitized when it was added to the
+            // address book
+            last_seen.saturating_elapsed() < constants::REACHABLE_PEER_DURATION
```

Comment:
Question: Can we use less than or equal here, to be consistent with the other `recent` methods?

If there's a reason for using strict less than, let's change all of the comparisons to strict less than, and add a comment.

Reply:
Done in [198e6a0](https://github.com/jvff/zebra/commit/198e6a0ead5653e678ba6bd4a12bd9d2e71a90d4), thanks! That was an oversight 😓